tests/omprog-feedback.sh

```diff
@@ -21,15 +21,14 @@ template(name="outfmt" type="string" string="%msg%\n")
         queue.type="Direct"  # the default; facilitates sync with the child process
         confirmMessages="on"
         useTransactions="off"
-        action.resumeRetryCount="10"
-        action.resumeInterval="1"
+        action.resumeInterval="1"  # retry interval: 1 second
+        action.resumeRetryCount="0" # the default; no need to increase since
```

Comment:
if the default is meant, it would be best to remove that parameter. This guards against default changes (that cause regeressions).

Reply:
OK. However in this particular case I prefer to leave the line commented-out (rather than delete it) for self-documentation purposes, ok? I have also left queue.type="Direct" (although this is the default) in the tests where this setting is relevant (to avoid someone changing it accidentally...).